core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -674,6 +677,18 @@ private[spark] class Executor(
     }
   }
 
+  private def setMDCForTask(taskDescription: TaskDescription): Unit = {
+    val properties = taskDescription.properties
+
+    org.slf4j.MDC.put("appId", properties.getProperty("spark.app.id"))
+    org.slf4j.MDC.put("appName", properties.getProperty("spark.app.name"))
```

Comment:
`taskDescription.properties` doesn't have these two attributes. We can try `taskDescription.name` firstly.

Reply:
@Ngone51  in debug I see it:
![image](https://user-images.githubusercontent.com/5586356/80370560-0c8a8580-8899-11ea-9632-0b9364986d68.png)

Reply:
hmm..they may need to be set explicitly? I didn't see them with default configurations.

Anyway, I'd suggest to use `taskDescription.name` firstly since this is a runnable for task and so we could track each task's logs.

Reply:
+1

Reply:
instead of appName? I am ok with that.

Reply:
`org.slf4j.MDC.put("taskName", taskDescription.name)`